src/frontend/src/components/RecordSearch/SearchPanel/index.tsx

```diff
@@ -75,6 +75,7 @@ export default function SearchPanel() {
       </div>
 
       {aliases.map((alias: AliasData, index: number) => {
+        console.log(index);
```

Comment:
Is this for the alias index bug? You want to get this logging into dev?

Reply:
Yes, I meant to do that temporarily for testing, but it didn't log anything. I forgot to put that on a different branch.